qulice-maven-plugin/src/test/java/com/qulice/maven/DefaultMavenEnvironmentTest.java

```diff
@@ -128,4 +128,30 @@ public final void passPathsWithWhitespaces()  throws Exception {
             Matchers.notNullValue()
         );
     }
+
+    /**
+     * DefaultMavenEnvironment can produce empty collection when no matches
+     * with checker.
+     * @throws Exception If something wrong happens inside
+     * @todo #676:30min Ideally integration test required to reproduce NPE.
```

Comment:
@mkordas You mean in puzzle description? I suppose it would be duplication - after @todo and then in description, don't you? I've looked at another puzzles - only one place of issue number. 
Also there is no issue number in test documentations in module qulice-maven-plugin, so I suppose this is not common practice.